doc/guides/using-internal-errors.md

```diff
@@ -107,7 +107,7 @@ message string. A simple example is:
 // Test ERR_TLS_CERT_ALTNAME_INVALID
 assert.strictEqual(
   errors.message('ERR_TLS_CERT_ALTNAME_INVALID', ['altname']),
-  'Hostname/IP does not match certificate\'s altnames: altname');
+  'Hostname/IP does not match certificate\'s altname: altname');
```

Comment:
I'm pretty sure this one is correct the way it is. A certificate can have multiple altnames.  Can you put just this one back the way it was? All the other ones look good to me.